HOST lewis.lbl.gov
only_i 202
only_code None
force_amber_prep True
dry_run False
... 202 1otj
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/ot/1otj
1otj.pdb 4phenix_1otj.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/ot/pdb1otj.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/ot/1otj/1otj.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/ot/pdb1otj.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/ot/1otj/1otj.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 1otj.pdb
===================================================

----------Chains
The following (original) chains have been found:
A
B
C
D

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
GLU_65
LEU_66
ARG_517
TRP_518
GLN_519
GLN_613
GLN_614
ARG_797
TRP_798
SER_857
GLY_858
ASP_932
ASN_933
GLU_974
ALA_975
GLU_1042
GLY_1043
ARG_1077
TRP_1078

---------- Mising heavy atom(s)

ARG_24 misses 7 heavy atom(s)
GLU_214 misses 4 heavy atom(s)
SER_222 misses 2 heavy atom(s)
SER_501 misses 2 heavy atom(s)
GLU_639 misses 4 heavy atom(s)
SER_781 misses 1 heavy atom(s)
GLU_1008 misses 4 heavy atom(s)
ARG_1013 misses 6 heavy atom(s)
GLU_1014 misses 4 heavy atom(s)
VAL_1016 misses 2 heavy atom(s)
SER_1061 misses 2 heavy atom(s)
The alternate coordinates have been discarded.
Only the first occurrence for each atom was kept.
not prmtop_filename /net/lewis/scratch1/nigel/amber/hi_lo/ot/1otj/4amber_1otj.prmtop
not prmtop_filename /net/chevy/raid1/nigel/amber/hi_lo/ot/1otj/4amber_1otj.prmtop

	Running AmberPrep : 1otj.pdb


==================================================
Running pdb4amber on 1otj.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================

  Residue "TAU" already in amber monomer library. Skipping elbow/antechamber
    run for this residue.
        

==================================================
Preparing asu files and 4phenix_1otj.pdb
==================================================

~> tleap -f 1otj_asu_tleap_input_run

Checking output filenames
  file : 1otj_asu.prmtop
Sorry:   Output file is empty : 1otj_asu.prmtop
  Check contents of "leap.log"
